python/cuml/naive_bayes/naive_bayes.py

```diff
@@ -436,6 +451,20 @@ def predict_log_proba(self, X) -> CumlArray:
         result = jll - log_prob_x.T
         return result
 
+    @staticmethod
+    def _convert_x_sparse(X):
```

Comment:
Does this need to be a static method? Since its a private method and not used outside of this file, it would be better suited as a normal function in the module (outside the class).

Reply:
Is there a benefit to making this an instance method when it doesn't require accessing or mutating any state on the instance? In general, I find it easier to read and refactor code that is explicit about this intention (being more of a helper function than an instance method). I was trying not to separate this from the class altogether but I can do that if you prefer.